ghost/i18n/locales/nl/ghost.json

```diff
@@ -1,25 +1,25 @@
 {
     "All the best!": "Tot snel!",
-    "Complete signup for {{siteTitle}}!": "Bevestig je registratie voor {{siteTitle}}!",
-    "Complete your sign up to {{siteTitle}}!": "Rond je registratie af voor {{siteTitle}}!",
+    "Complete signup for {{siteTitle}}!": "Bevestig je aanmelding voor {{siteTitle}}!",
+    "Complete your sign up to {{siteTitle}}!": "Rond je aanmedling af voor {{siteTitle}}!",
     "Confirm email address": "Bevestig e-mailadres",
     "Confirm signup": "Bevestig registratie",
     "Confirm your email address": "Bevestig je e-mailadres",
     "Confirm your email update for {{siteTitle}}!": "Bevestig je nieuwe e-mailadres voor {{siteTitle}}!",
     "Confirm your subscription to {{siteTitle}}": "Bevestig je abonnement op {{siteTitle}}",
     "For your security, the link will expire in 24 hours time.": "Voor je veiligheid vervalt de link na 24 uur.",
-    "Hey there,": "Hey,",
-    "Hey there!": "Hey!",
-    "If you did not make this request, you can safely ignore this email.": "Als deze aanvraag niet van jou is, kun je deze e-mail negeren.",
+    "Hey there,": "Hallo,",
+    "Hey there!": "Hallo!",
+    "If you did not make this request, you can safely ignore this email.": "Als deze aanvraag niet van jou is, kun je deze e-mail veilig negeren.",
```

Comment:
Despite this edit being more "correct" in a literal sense, this sounds somewhat odd in Dutch so we'd like to leave out the word "veilig" here.